maps/expert/dark_castle/chimps.ahk

```diff
@@ -0,0 +1,135 @@
+DarkCastleChimps() {
+    global TS := Map(
+        "Obyn",["hero",[865,447]],
+        "Dart A",["dart",[739,656]],
+        "Dart B",["dart",[688,691]],
+        "Sub A",["sub",[1083,690]],
+        "Sub B",["sub",[1086,398]],
+        "Alch A",["alch",[1011,657]],
+        "Alch B",["alch",[666,388]],
+        "Alch C",["alch",[711,346]],
+        "Druid A",["druid",[715,433]],
+        "Druid B",["druid",[789,447]],
+        "Druid C",["druid",[941,448]],
+        "Druid D",["druid",[766,384]],
+        "Druid E",["druid",[842,384]],
+        "Druid F",["druid",[918,385]],
+        "Spike",["spike",[1022,441]],
+        "Village",["village",[869,303]]
+    )
+
+    Place("Dart A")
+    Place("Sub A")
+
+    StartGame()                     ; Round 6
+
+    WaitForRound(7)
+    Place("Dart B")
+
+    WaitForRound(10)
+    Place("Obyn",true)
+
+    WaitForRound(11)
+    Upgrade("Sub A",1,0,0)          ; 000 -> 100
+
+    WaitForRound(14)
+    Upgrade("Sub A",1,0,0)          ; 100 -> 200
+
+    WaitForRound(15)
+    Upgrade("Sub A",0,0,1,true)     ; 200 -> 201
+
+    WaitForRound(19)
+    Upgrade("Sub A",0,0,1,true)     ; 201 -> 202
+
+    WaitForRound(20)
+    Upgrade("Dart A",0,0,2,true)    ; 000 -> 002
+
+    WaitForRound(25)
+    Upgrade("Sub A",0,0,1)          ; 202 -> 203
+
+    WaitForRound(27)
+    Place("Alch A")
+
+    WaitForRound(28)
+    Upgrade("Alch A",2,0,0)         ; 000 -> 200
+
+    WaitForRound(32)
+    Upgrade("Alch A",1,0,0)         ; Alch 1: 200 -> 300

```

Comment:
Tower to upgrade mentioned in the comment even though it's already readable from the function call

Reply:
removed